### ISSUE TITLE
NS-430 values out of range

### ISSUE THREAD

**Issue by jdahlblom** [DCS-Skunkworks/dcs-bios#412]
Observed from F-14.
~~~
2023-10-13 17:05:12  ERROR  MemoryAllocation.lua: value -3761.000000 is too small for NS430:NS430_VOL_PUSH_ID_VOL (address 1540 mask 65535)

NS430:definePotentiometer("NS430_VOL_PUSH_ID_VOL", 257, 3005, 2, { 0, 1 }, "NS430", "VOL_PUSH-ID VLOC Volume")

~~~

Above value from DCS is ```-0.057379841804504```

Also :
```
2023-10-13 17:16:22  ERROR  MemoryAllocation.lua: value -1.000000 is too small for NS430:NS430_PWR_VOL_PUSH_SQ_POW (address 1536 mask 1)

NS430:defineDoubleCommandButton("NS430_PWR_VOL_PUSH_SQ_POW", 257, 3001, 3030, 1, "NS430", "PWR-VOL PUSH-SQ COM Power")
```
Above value from DCS is ```-0.006003350019455```

**Comment by charliefoxtwo** [comment]
> Above value from DCS is `-0.006003350019455`

given this, let's maybe modify the epsilon in MemoryAllocation to just be 0.01? That should still be sufficiently small that it won't mask actual errors, but large enough it should help with some of the other controls we're seeing with tight ranges (like in the Mi-8)
https://github.com/DCS-Skunkworks/dcs-bios/blob/a7a307f25f9d062f2cb1047c9408eb2abfdde8ea/Scripts/DCS-BIOS/lib/modules/memory_map/MemoryAllocation.lua#L54

**Comment by jdahlblom** [comment]
> > Above value from DCS is `-0.006003350019455`
> 
> given this, let's maybe modify the epsilon in MemoryAllocation to just be 0.01? That should still be sufficiently small that it won't mask actual errors, but large enough it should help with some of the other controls we're seeing with tight ranges (like in the Mi-8)
> 
> https://github.com/DCS-Skunkworks/dcs-bios/blob/a7a307f25f9d062f2cb1047c9408eb2abfdde8ea/Scripts/DCS-BIOS/lib/modules/memory_map/MemoryAllocation.lua#L54

OK

**Comment by charliefoxtwo** [comment]
@jdahlblom was this fixed in #461?

**Comment by jdahlblom** [comment]
Yes, closing this one. Just double-checked.
